src/entry.js

```diff
@@ -4,16 +4,14 @@ import SlideDeck from './index'
 
 const mod = require(DOC_FILENAME)
 const slides = mod.default
-const { theme, components, Provider } = mod
```

Comment:
This looks like it'd mean a breaking change – what do you think of keeping backwards compatibility even if it's not documented?

Reply:
See what you mean. Then let's keep it.